.github/workflows/c1_c2_c3_app_release_restricted.yml

```diff
@@ -13,6 +13,10 @@ on:
           - all
         required: true
         default: 'all'
+      countly_enabled:
+        description: 'Enable Countly'
+        type: boolean
+        default: false
```

Comment:
don't expose the option and use false

Reply:
fixed: 50b512451112562644363c5711c8bd14b3c0c2dd